src/pages/VisualizationPage.js

```diff
@@ -1,36 +1,60 @@
 import React from 'react';
 import styled from 'styled-components';
 import PropTypes from 'prop-types';
-import Typography from '@material-ui/core/Typography';
+import { Paper, Tabs, Tab } from '@material-ui/core';
 import BalanceHistory from '../charts/BalanceHistory';
 import PartyGrouping from '../charts/PartyGrouping';
 import CategoryGrouping from '../charts/CategoryGrouping';
+import TransactionHistory from '../charts/TransactionHistory';
 
-const Container = styled.div`
-  text-align: center;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  max-width: 60rem;
-  margin: 3rem auto;
+const TabContainer = styled(Paper)`
+  flex-grow: 1;
 `;
 
-const Visualization = ({ initialTransactions, categories }) => (
-  <Container>
-    <Typography variant="h2">Visualization</Typography>
-    {!initialTransactions ? (
-      <Typography variant="h4">Select a file and bank on import page</Typography>
-    ) : (
+class VisualizationPage extends React.Component {
+  state = { value: 0 };
+
+  handleChange = (event, value) => {
+    this.setState({ value });
+  };
+
+  getChart = () => {
+    const { initialTransactions, categories } = this.props;
+    const { value } = this.state;
+    if (value === 0) return <TransactionHistory initialTransactions={initialTransactions} />;
```

Comment:
Suggestion: I would assign the numbers to a const variables and then do the comparison against the variable instead to improve readability of the code and the ease of modifying  of different steps.